src/features/unified-share-modal/constants.js

```diff
@@ -23,21 +31,61 @@ const COLLAB_GROUP_TYPE = 'group';
 const COLLAB_USER_TYPE = 'user';
 const COLLAB_PENDING_TYPE = 'pending';
 
+/**
+ * The following constants are used for converting API requests
+ * and responses into objects expected by the USM, and vice versa
+ */
+
+const API_TO_USM_ACCESS_LEVEL_MAP = {
+    [ACCESS_COLLAB]: PEOPLE_IN_ITEM,
```

Comment:
nit: I know you created some data conversion functions in previous commits. Perhaps instead of mapping here, we should define the API constants only, and keep the conversions together by putting the maps with the converter functions?